package.json

```diff
@@ -68,7 +68,7 @@
   },
   "peerDependencies": {
     "@edx/frontend-platform": "^1.8.0",
-    "@edx/paragon": ">= ^7.0.0 < 13.0.0",
+    "@edx/paragon": ">= ^7.0.0 < 17.0.0",
```

Comment:
For what it's worth, I believe Paragon is up to v19.  If it's not a pain, we may want to jump all the way up to latest...

Reply:
Actually we've a [peer dependency](https://github.com/openedx/frontend-platform/blob/ee67eec81511575f1177dbb90e121dc20d65caa6/package.json#L73) in frontend-platform which restricts the paragon version to <17.0.0 and raises conflict if I upgrade paragon version to 19 here and run `npm update`. We're working on releasing the new paragon version with Node 16 support. Once it is out, I'll upgrade to the new paragon version everywhere.